pyIsis/connection.py

```diff
@@ -36,21 +36,23 @@ class Client(object):
         >>> client.get_workspaces()
     """
 
-    def __init__(self, hostname, username, password):
+    def __init__(self, hostname, username, password, port=80):
```

Comment:
use `ISIS_SOAP_PORT` global variable instead of value

Reply:
Corrected in [last commit ](https://github.com/predat/pyIsis/pull/6/commits/5674cd66df066800e098aba294533317f38ba012)